src/test/java/com/hubspot/jackson/jaxrs/PropertyFilterTest.java

```diff
@@ -60,14 +61,74 @@ public void testNestedExclude() {
     assertThat(node.get("propB").has("key2")).isFalse();
   }
 
+  @Test
+  public void testWildcardNestedInclude() {
+    JsonNode input = node().set("propC", mapper.createObjectNode().put("key3", "value3"));
+    JsonNode node = filter(input, "*.key1");
+
+    assertThat(node.get("propA").get("key1").textValue()).isEqualTo("value1");
+    assertThat(node.get("propA").has("key2")).isFalse();
+    assertThat(node.get("propB").get("key1").textValue()).isEqualTo("value1");
+    assertThat(node.get("propB").has("key2")).isFalse();
+    assertThat(node.get("propC")).hasSize(0);
+  }
+
+  @Test
+  public void testWildcardNestedExclude() {
+    JsonNode node = filter("!*.key2");
+
+    assertThat(node.get("propA").get("key1").textValue()).isEqualTo("value1");
+    assertThat(node.get("propA").has("key2")).isFalse();
+    assertThat(node.get("propB").get("key1").textValue()).isEqualTo("value1");
+    assertThat(node.get("propB").has("key2")).isFalse();
+  }
+
+  @Test
+  public void testReadmeWildcardInclusion() throws Exception {
+    String json = "{" +
+        "  \"id\": 54," +
+        "  \"name\": \"Object\"," +
+        "  \"child\": {" +
+        "    \"id\": 96," +
+        "    \"name\": \"Child Object\"" +
+        "  }" +
+        "}";
```

Comment:
I find this style of creating test input JSON to be clearer & less verbose than programmatically creating `ObjectNode`s.